src/library_sdl.js

```diff
@@ -348,13 +348,13 @@ var LibrarySDL = {
     translateRGBAToColor: (r, g, b, a) => r | g << 8 | b << 16 | a << 24,
 
     makeSurface(width, height, flags, usePageCanvas, source, rmask, gmask, bmask, amask) {
-      var is_SDL_HWSURFACE = flags & 0x00000001;
```

Comment:
It looks like `SDL_HWSURFACE` constant was wrong here.  It should have been `0x08000001`.   Luckily it was still matching because its a subset of the bits.